openkfw/TruBudget
# 1906-show all tags

### Checklist




### Description

* added popover that shows the rest of the tags in workflow items

### How to test

1. log into the app and open any project and its subproject
2. add tags to any workflow item where possible and after submission check outcome on workflow item page
3. try different possibilities like: add only one tag - expand button should not be present 
4. try adding several tags - expand button should be present besides the tag
5. after clicking expand button, all tags should be visible in popover




Closes #1906
